apps/test/solutions/applab/ec_design.js

```diff
@@ -593,6 +591,79 @@ module.exports = {
     },
 
     {
+      description: "exercise duplicate button on elements",
+      editCode: true,
+      xml: '',
+      runBeforeClick: function (assert) {
+
+        // Switch to design mode
+        var designModeButton = $('#designModeButton');
+        designModeButton.click();
+
+        // Add an image
+        testUtils.dragToVisualization('IMAGE', 0, 0);
+        var designModeViz = $('#designModeViz');
+        var newImage = designModeViz.find('img');
+        assert.equal(newImage.length, 1);
+
+        // Duplicate the image
+        var imageDuplicateButton = $('#designWorkspaceBody').find('button:contains(Duplicate)')[0];
+        ReactTestUtils.Simulate.click(imageDuplicateButton);
+        assert.equal(designModeViz.find('img').length, 2);
```

Comment:
Nice! Please also assert that the position and element id of the new image and chart, since those seem easy to test and also easy for us to accidentally break in the future